sds/src/scanner/mod.rs

```diff
@@ -1094,6 +1095,52 @@ mod test {
         assert_eq!(content, "[IDCARD] 513231200012121651");
     }
 
+    #[test]
+    fn test_iban_checksum() {
```

Comment:
As for NHS checksum, this can be simplified a bit to validate that the iban/bban are validated when `IbanChecker` is supplied.